### DISCUSSION TITLE
Enable Digital Music by Default if Audio Pack is Downloaded?

### DISCUSSION THREAD

**Discussion by KnightCaptainU7** [exult/exult#589] [General]
Just did a clean install on a new computer, and I selected all options for the installation, which includes the Exult Audio pack. But it's not enabled by default. Might this be a worthwhile change for a Feature Request?

I love the new installer by the way.

**Comment by DominusExult** [comment]
We could make Digital Music the default when the oggs are there but I'm not sure that's a good idea. FMOpl is used for default since that is the lowest hanging fruit, no need for a Midi device etc... Ogg should be fine but who knows :)
Also people are more familiar with the FMOpl music as that's what people had back then. Almost no one had an MT32 device.
